Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# PrÃ¡ctica 1 

n = 81
a = 0.
b = 3.*np.pi/2.
x_i = np.linspace(a, b, n)
dx = (b-a)/(n-1)

# Defino la matriz de derivaciÃ³n D





# Condiciones de contorno




### RepresentaciÃ³n grÃ¡fica

------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
Cell [0;32mIn[1], line 5[0m
[1;32m      3[0m n [38;5;241m=[39m [38;5;241m81[39m
[1;32m      4[0m a [38;5;241m=[39m [38;5;241m0.[39m
[0;32m----> 5[0m b [38;5;241m=[39m [38;5;241m3.[39m[38;5;241m*[39m[43mnp[49m[38;5;241m.[39mpi[38;5;241m/[39m[38;5;241m2.[39m
[1;32m      6[0m x_i [38;5;241m=[39m np[38;5;241m.[39mlinspace(a, b, n)
[1;32m      7[0m dx [38;5;241m=[39m (b[38;5;241m-[39ma)[38;5;241m/[39m(n[38;5;241m-[39m[38;5;241m1[39m)

[0;31mNameError[0m: name 'np' is not defined
NameError: name 'np' is not defined

